### PR TITLE
[author] Update package.json for qoopido.js

### PR DESCRIPTION
**Pull request by dlueth**
- add git autoupdate section

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
I think we should remove the part of npm auto-update config.

**Comment by dlueth** [comment]
Removed the obsolete npm settings - thanks for the feedback!

**Comment by PeterDaveHello** [comment]
Would you please squash them into only one commit? Thanks.

File: ajax/libs/qoopido.js/package.json [hunk 1]
```diff
@@ -34,13 +33,13 @@
...
-      "files": [
-        "**"
-      ]
-    }
-  ],
-  "filename": "base.js"
+  "filename": "base.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/dlueth/qoopido.js.git",
```
Comment (PeterDaveHello) [inline ajax/libs/qoopido.js/package.json]: Should use git protocol here.

**Comment by dlueth** [comment]
Argh, messed up the squash commit again - will close this pull request and start a new one from scratch...

**Pull request closed without merging**